get_the_books.sh

```diff
@@ -0,0 +1,7 @@
+for text_ID in $(cat Text_IDs.csv); do
```

Comment:
super combo here with the file with the book ids and a script to get them.  It's totally scalable and efficient.  I've often done this on short scripts and once in a while I next the make calls.  In this case it's totally clear, so just as an fyi, if you get another level of calling make, it's probably worth a rewrite to use direct bash.  Awesome work!